src/lib.rs

```diff
@@ -555,7 +561,7 @@ macro_rules! mimes {
 
         #[test]
         fn test_mimes_macro_consts() {
-            [
+            let _ = [
```

Comment:
Why did you put this assignment here, just curious?

Reply:
It fixed an "unused return value" error in the tests. That's why some of the other pull requests are failing CI, too.